Tasks/AzurePowerShellV2/task.json

```diff
@@ -135,7 +135,7 @@
             "defaultValue": "",
             "required": true,
             "visibleRule": "TargetAzurePs = OtherVersion",
-            "helpMarkDown": "Preferred Azure PowerShell Version needs to be a proper semantic version eg. 1.2.3. Regex like 2.\\*,2.3.\\* is not supported. The Hosted VS2017 Pool currently supports versions: 2.1.0, 3.8.0, 4.2.1, 5.1.1"
+            "helpMarkDown": "Preferred Azure PowerShell Version needs to be a proper semantic version eg. 1.2.3. Regex like 2.\\*,2.3.\\* is not supported. The Hosted VS2017 Pool currently supports Azure module versions: 2.1.0, 3.8.0, 4.2.1, 5.1.1 and AzureRM module versions: 2.1.0, 3.8.0, 4.2.1, 5.1.1, 6.7.0"
```

Comment:
Instead of Azure Module, just say "the Az Module" coz I believe thats what is used in all the documentation: https://docs.microsoft.com/en-us/powershell/azure/new-azureps-module-az

Reply:
Also please make this change wherever required.

Reply:
AzurePowerShellV2 and V3 supported Azure and AzureRm modules only. Az module support was added first in V4.